### ISSUE TITLE
[Pricing Table]: QA: Entire page is not visible to the user at 400% zoom

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#9159]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/ADCMS-2457
- DPO consulting engineer: N/A

-----

### Description

Page element:
Entire page

Issue : When the page is zoomed to 400%, entire page is not visible to the user.

Expected: Entire page should be visible even when the page is zoomed to 400%.

User Impact: When the page is not visible on 400% zoom, user with low vision will not be able to see the entire page content.

### Component(s) impacted

Pricing Table

### Browser

Firefox

### Carbon for IBM.com version

v1.36.0-rc.0

### Severity

Severity 2 = Aspects of design is broken, and impedes users in a significant way, but there is a way to complete their tasks. Affects major functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by jkaeser** [comment]
This may be a Storybook issue as much as it is an issue with the Pricing Table. You can see the table text is completely hidden within the context of the Storybook UI:
<img width="720" alt="Screen Shot 2022-07-25 at 3 02 16 PM" src="https://user-images.githubusercontent.com/12532727/180857095-f05dc68d-ef93-4edc-b52a-664a1e5db935.png">

Outside of the Storybook UI it looks like this:
<img width="720" alt="Screen Shot 2022-07-25 at 3 23 42 PM" src="https://user-images.githubusercontent.com/12532727/180858243-8c533161-e764-4a11-a11d-fd298bcaec8e.png">

**Comment by oliviaflory** [comment]
@jkaeser I believe the concern is that user is unable to see the tags, description and list items at 400% zoom because the sticky header is triggered very quickly due to the zoom. 
<img width="1201" alt="Screen Shot 2022-07-25 at 4 26 38 PM" src="https://user-images.githubusercontent.com/45692622/181077514-e058bd07-522c-44bf-9a4e-22cb86013446.png">


![Pricing table 400](https://user-images.githubusercontent.com/45692622/181077698-a4f56ae5-aba9-4842-98ac-6b58633ed6af.gif)

**Comment by gopinathan-n** [comment]
@jkaeser @kennylam Verified in https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental/iframe.html?id=components-pricing-table--default
Still able to replicate the issue.

**Comment by jkaeser** [comment]
> @jkaeser @kennylam Verified in https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components-experimental/iframe.html?id=components-pricing-table--default Still able to replicate the issue.

@gopinathan-n You can test the fix for this issue in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9205#issuecomment-1210632691.
